snap/snapcraft.yaml

```diff
@@ -83,6 +83,12 @@ parts:
     source: build-scripts
     override-build: ./build-component.sh helm
 
+  gateway-api-crds:
+    after: [build-deps, helm]
+    plugin: nil
+    source: build-scripts
+    override-build: ./build-component.sh gateway-api-crds
+
```

Comment:
let's rather not do this at build time, better to just have the script around and have the chart checked in the repository.